repositories/src/test/scala/cromwell/pipeline/datastorage/dao/utils/PostgreTablesCleaner.scala

```diff
@@ -0,0 +1,18 @@
+package cromwell.pipeline.datastorage.dao.utils
+
+import cromwell.pipeline.datastorage.DatastorageModule
+import org.scalatest.{ BeforeAndAfterEach, Suite }
+
+trait PostgreTablesCleaner extends Suite with BeforeAndAfterEach {
```

Comment:
BeforeAndAfterEach has a self-type
this : org.scalatest.BeforeAndAfterEach with org.scalatest.Suite

I don't know how to avoid extending without Suite in that case. Would appreciate any advice.

Reply:
You can make this trait self-typed too

Reply:
Like this? Fixed